includes/HelpCenterFeatureHooks.php

```diff
@@ -37,8 +64,14 @@ public function hooks() {
 	 * @return boolean the filtered value
 	 */
 	public function filterValue( $value ) {
+		$helpCenterFeature = Features::getInstance()->getFeature( 'helpCenter' );
```

Comment:
Maybe use the already-defined variable to keep everything consistent?
https://github.com/newfold-labs/wp-module-help-center/blob/a09d1cf6bf5c95bc93d79842f8c46ce3472caef2/includes/HelpCenterFeature.php#L23